tests/common/fixtures/ptfhost_utils.py

```diff
@@ -193,3 +198,37 @@ def run_icmp_responder(duthost, ptfhost, tbinfo):
 
     logging.debug("Stop running icmp_responder")
     ptfhost.shell("supervisorctl stop icmp_responder")
+
+
+@pytest.fixture(scope='session', autouse=True)
+def run_garp_service(duthost, ptfhost, tbinfo):
```

Comment:
Do we need to ensure each interface on PTF has unique MAC address (```change_mac_addresses```) before this fixture?

Reply:
I think this would be a good idea. Is there any negative effects caused by running the `change_mac` script?

Reply:
No. I don't think running ```change_mac``` would cause nagative effects. Actually, each interface on ptf already has unique MAC address after deploying. Running ```change_mac``` is a double-check, I think.